docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,233 @@
+# Deploy Scalar DL on AWS
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker.
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability.
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Amazon Web Services (AWS) as a starting point for deploying Scalar DL for production.
+
+## What we create
+
+![image](images/network_diagram_eks.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service
+    * DynamoDB
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards. Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production.
+This section shows how to configure a secure network for Scalar DL deployments.
+
+### Requirements
+ 
+* You must create VPC with NAT gateways on private networks. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
```

Comment:
it's better to use `Public NAT Gateway', Because it is now possible to create `Private NAT Gateway`.

https://aws.amazon.com/about-aws/whats-new/2021/06/aws-removes-nat-gateways-dependence-on-internet-gateway-for-private-communications/

Reply:
Thank you, updated in b1fc1b7